pkg/cableengine/cableengine.go

```diff
@@ -178,24 +187,25 @@ func (i *engine) installCableWithNATInfo(rnat *natdiscovery.NATEndpointInfo) err
 
 	for j := range activeConnections {
 		active := &activeConnections[j]
-		logger.V(log.TRACE).Infof("Analyzing currently active connection %q", active.Endpoint.CableName)
+		activeFamilyCableName := active.Endpoint.GetFamilyCableName(rnat.UseFamily)
+		logger.V(log.TRACE).Infof("Analyzing currently active connection %q", activeFamilyCableName)
```

Comment:
It's only accurate to log the cable name using `rnat.UseFamily` if it matches `active.GetFamily()`.  We should use the latter here.

```suggestion
                activeFamily = active.GetFamily()
		activeFamilyCableName := active.Endpoint.GetFamilyCableName(activeFamily)
```